### PR TITLE
rapper: add page

### PR DESCRIPTION
**Pull request by fkraeutli**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/rapper.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# rapper
+
+> RDF parser.
+> Part of the Raptor RDF Syntax Library.
+> Homepage: <http://librdf.org/raptor/rapper.html>.
```
Comment (pxgamer) [inline pages/common/rapper.md]:
```suggestion
> More information: <http://librdf.org/raptor/rapper.html>.
```

File: pages/common/rapper.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# rapper
+
+> RDF parser.
```
Comment (pxgamer) [inline pages/common/rapper.md]:
```suggestion
> The Raptor RDF parsing utility.
```

**Comment by sbrl** [comment]
Erm I think you've gotten yourself a bit confused, @stale. This has already been merged....

**Comment by bl-ue** [comment]
L O L !!!

**Pull request merged**